packages/core/src/App/Containers/RealAccountSignup/change-account-currency.jsx

```diff
@@ -36,6 +45,28 @@ const ChangeAccountCurrency = ({ legal_allowed_currencies, value, onSubmit, form
                     <Text as='h3' size='xxs' align='center' className='change-currency__sub-title'>
                         <Localize i18n_default_text='Choose the currency you would like to trade with.' />
                     </Text>
+                    {!can_change_fiat_currency && (
+                        <div className='account-wizard--disabled-message'>
+                            <p>
```

Comment:
Instead of the `<p>` tag, try `<Text as='p' ...>` 🙏

Reply:
Done